cpu/cc26x0/include/cc26x0_rfc.h

```diff
@@ -393,17 +403,39 @@ typedef struct {
 
 
 typedef struct {
-    radio_op_command_t op;
-    uint8_t mode;
+    uint16_t commandNo; /* W */
```

Comment:
@yogo1212 
using the `radio_op_command_s` directly will add a memory overhead inside ` radio_setup_cmd`. that might be one of the reason `ble_setup()` isn't working right now...